src/main/kotlin/no/nav/familie/ks/sak/integrasjon/sanity/domene/SanityBegrunnelse.kt

```diff
@@ -10,21 +10,30 @@ data class SanityBegrunnelse(
     val apiNavn: String?,
     val navnISystem: String,
     val vilkaar: List<Vilkår>? = null,
-    val rolle: List<PersonType> = emptyList()
+    val rolle: List<PersonType> = emptyList(),
+    val endringsaarsaker: List<Årsak>? = null,
+    val ovrigeTriggere: List<ØvrigTrigger>? = null,
+    val lovligOppholdTriggere: List<VilkårTrigger>? = null,
+    val bosattIRiketTriggere: List<VilkårTrigger>? = null,
+    val borMedSokerTriggere: List<VilkårTrigger>? = null,
+    val endretUtbetalingsperiodeTriggere: List<EndretUtbetalingsperiodeTrigger>? = null,
+    val endretUtbetalingsperiodeDeltBostedUtbetalingTrigger: EndretUtbetalingsperiodeDeltBostedTriggere? = null,
```

Comment:
Mulig vi skal ha med disse feltene for KS også så gjør jo ikke noe om de legges inn nå, men det kan godt hende at de ikke skal være med også. Gjenstår litt for å finne ut hvilke vi skal beholde og hvor "komplisert" vi trenger å gjøre begrunnelser for KS.